### PR TITLE
feat: Sort Contacts via name, email, phone_number, last_activity

### PR DESCRIPTION
**Pull request by pranavrajs**
- Sort Contacts via name, email, phone_number, last_activity_at

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/contacts/components/ContactsView.vue [hunk 1]
```diff
@@ -87,37 +90,55 @@ export default {
...
   },
   methods: {
+    updatePageParam(page) {
+      window.history.pushState({}, null, `${this.$route.path}?page=${page}`);
+    },
+    getSortAttribute() {
+      let sortAttr = Object.keys(this.sortConfig).reduce((acc, sortKey) => {
+        const direction = this.sortConfig[sortKey];
+        if (direction) {
+          acc += `${direction === 'asc' ? '' : '-'}${sortKey}`;
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/contacts/components/ContactsView.vue:103]: Can we simplify with by taking the interpolation outside?

File: app/javascript/dashboard/routes/dashboard/contacts/components/ContactsView.vue [hunk 2]
```diff
@@ -87,37 +90,55 @@ export default {
...
+        });
+      }
+    },
     onInputSearch(event) {
       const newQuery = event.target.value;
       const refetchAllContacts = !!this.searchQuery && newQuery === '';
+      this.searchQuery = newQuery;
       if (refetchAllContacts) {
-        this.$store.dispatch('contacts/get', { page: 1 });
+        this.fetchContacts(1);
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/contacts/components/ContactsView.vue:130]: If `1` id the page index? Can use use `const DEFAULT_PAGE_INDEX` ?

**Review by nithindavid — CHANGES_REQUESTED** [review]
Left some minor comments. Please see them.

**Review by sojan-official — DISMISSED** [review]
LGTM

- backend changes
- swagger

**Review by sojan-official — APPROVED** [review]

**Pull request merged**